apps/wq_hypersweep/resnet.py

```diff
@@ -0,0 +1,139 @@
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+import numpy as np
+import datetime as dt
+import os.path
+from os import path
+import sys
+import getopt
+
+
+# Default hyperparamter values
+BATCHSIZE = 64
+RESNETBLOCKS = 10
+DROPOUTRATE = 0.5
+EPOCHS = 30
+EPOCHSTEPS = 195
+VALIDATIONSTEPS = 3 
+
+# Command line changes to hyperparams
+try:
+	opts, args = getopt.getopt(sys.argv[1:],"hb:r:d:e:s:v",[])
+except getopt.GetoptError:
+	print('usage: test.py -b <batch size> -r <number of ResNet blocks> -d <dropout rate> -e <number of epochs> -s <steps per epoch> -v <validation steps>')
+	sys.exit(2)
+for opt, arg in opts:
+	if opt == '-h':
+		print('usage: test.py -b <batch size> -r <number of ResNet blocks> -d <dropout rate> -e <number of epochs> -s <steps per epoch> -v <validation steps>')
+		sys.exit()
+	elif opt in ("-b"):
+		BATCHSIZE = int(arg)
+	elif opt in ("-r"):
+		RESNETBLOCKS = int(arg)
+	elif opt in ("-d"):
+		DROPOUTRATE = float(arg)
+	elif opt in ("-e"):
+		EPOCHS = int(arg)
+	elif opt in ("-s"):
+		EPOCHSTEPS = int(arg)
+	elif opt in ("-v"):
+		VALIDATIONSTEPS = int(arg)
+
+
+#LOG_DIR = './log'
+#get_ipython().system_raw(
+#    'tensorboard --logdir {} --host 0.0.0.0 --port 6006 &'
+#    .format(LOG_DIR)
+#)
+
+
+# retrieve dataset
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.cifar10.load_data()
```

Comment:
Is this where the download is actually happening?
If so, I think you will want to download it once at the beginning of the sweep and pass it in as a file to be transferred...

Reply:
That is correct. I'll change this section to reflect that suggestion when I distribute the workload.